libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.h

```diff
@@ -338,14 +338,14 @@ extern NSString * const kSFAuthenticationManagerFinishedNotification;
  Kick off the login process for the given user.
  @param completionBlock The block of code to execute when the authentication process successfully completes.
  @param failureBlock The block of code to execute when the authentication process has a fatal failure.
- @param account User account to be logged in.
+ @param credentials SFOAuthCredentials to be logged in.
```

Comment:
NB: We need to change the description on this and `loginWithCompletion:failure:`, as they're no longer accurate.